src/test/resources/stubs/java/Z_mockito-ReturnsArgumentAt-answer.json

```diff
@@ -12,6 +12,7 @@
     "5a2415278e865ac0becfcbaf8626d4f0c3b17a70": "Yfilerename",
     "4b81c5d36e1ab63412c9a55d982cf5c6df238dab": "Ymultichange(Yfilerename,Ybodychange)",
     "dda1c41ea0fe96abfe62bb1bfd1e1d51d2c5c509": "Ybodychange",
-    "5490d9b2fc7814bc4c1efe5a66f05ce5874d38bc": "Ymultichange(Ymovefromfile,Yreturntypechange,Ybodychange)"
+    "5490d9b2fc7814bc4c1efe5a66f05ce5874d38bc": "Ymultichange(Ymovefromfile,Yreturntypechange,Ybodychange)",
+    "ae7ad39c99ea7ffecd84c1453aa26f44846f066a": "Yintroduced"
```

Comment:
This was really just missing the introduction commit. I checked it and it's all correct.